features/date.yml.dist

```diff
@@ -0,0 +1,185 @@
+# Generated from: date.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: high
+  baseline_low_date: 2015-07-29
+  baseline_high_date: 2018-01-29
+  support:
+    chrome: "1"
+    chrome_android: "18"
+    edge: "12"
+    firefox: "1"
+    firefox_android: "4"
+    safari: "1"
+    safari_ios: "1"
+compat_features:
+  # ⬇️ Same status as overall feature ⬇️
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "1"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "1"
+  #   firefox_android: "4"
+  #   safari: "1"
+  #   safari_ios: "1"
+  - javascript.builtins.Date
+  - javascript.builtins.Date.Date
+  - javascript.builtins.Date.UTC
+  - javascript.builtins.Date.getDate
+  - javascript.builtins.Date.getDay
+  - javascript.builtins.Date.getFullYear
+  - javascript.builtins.Date.getHours
+  - javascript.builtins.Date.getMilliseconds
+  - javascript.builtins.Date.getMinutes
+  - javascript.builtins.Date.getMonth
+  - javascript.builtins.Date.getSeconds
+  - javascript.builtins.Date.getTime
+  - javascript.builtins.Date.getTimezoneOffset
+  - javascript.builtins.Date.getUTCDate
+  - javascript.builtins.Date.getUTCDay
+  - javascript.builtins.Date.getUTCFullYear
+  - javascript.builtins.Date.getUTCHours
+  - javascript.builtins.Date.getUTCMilliseconds
+  - javascript.builtins.Date.getUTCMinutes
+  - javascript.builtins.Date.getUTCMonth
+  - javascript.builtins.Date.getUTCSeconds
+  - javascript.builtins.Date.parse
+  - javascript.builtins.Date.setDate
+  - javascript.builtins.Date.setFullYear
+  - javascript.builtins.Date.setHours
+  - javascript.builtins.Date.setMilliseconds
+  - javascript.builtins.Date.setMinutes
+  - javascript.builtins.Date.setMonth
+  - javascript.builtins.Date.setSeconds
+  - javascript.builtins.Date.setTime
+  - javascript.builtins.Date.setUTCDate
+  - javascript.builtins.Date.setUTCFullYear
+  - javascript.builtins.Date.setUTCHours
+  - javascript.builtins.Date.setUTCMilliseconds
+  - javascript.builtins.Date.setUTCMinutes
+  - javascript.builtins.Date.setUTCMonth
+  - javascript.builtins.Date.setUTCSeconds
+  - javascript.builtins.Date.toDateString
+  - javascript.builtins.Date.toLocaleDateString
+  - javascript.builtins.Date.toLocaleString
+  - javascript.builtins.Date.toLocaleTimeString
+  - javascript.builtins.Date.toString
+  - javascript.builtins.Date.toTimeString
+  - javascript.builtins.Date.toUTCString
+  - javascript.builtins.Date.valueOf
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "1"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "1"
+  #   firefox_android: "4"
+  #   safari: "4"
```

Comment:
This surprised me since I thought `Date.now()` was prehistoric, but it seems right:
https://github.com/WebKit/WebKit/commit/c6b29b42c1efbfe6507323ab0d7ba3eed6096d6d